ext/scummvm_log_download.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Now that Python code exists, shell scripts and files containing URLs can be removed.